mix.exs

```diff
@@ -38,6 +38,7 @@ defmodule Rig.Mixfile do
   def application do
     [
       mod: {Rig.Application, []},
+      included_applications: [:peerage],
```

Comment:
is there a specific reason for including this explicitly?

Reply:
Didn't find any other working way, how to include `peerage` library, but don't start it by default (so it's started only if needed).

Reply:
👍